numba/tests/test_np_functions.py

```diff
@@ -2448,6 +2452,15 @@ def a_variations():
 
         self.assertIn("Cannot np.flip on UniTuple", str(raises.exception))
 
+    def test_logspace_basic(self):
+        pyfunc = logspace
+        cfunc = jit(nopython=True)(pyfunc)
+
+        self.assertPreciseEqual(pyfunc(0, 6), cfunc(0, 6))
+        self.assertPreciseEqual(pyfunc(0, 6, num=100), cfunc(0, 6, num=100))
+        self.assertPreciseEqual(pyfunc(0, 6, base=2), cfunc(0, 6, base=2))
+        self.assertPreciseEqual(pyfunc(0, 6, 100, 2), cfunc(0, 6, 100, 2))
```

Comment:
It'd be good to add some more tests, perhaps consider what is permitted as input into the function and try some variations in value and type. One slightly unusual example might be:
```
In [14]: np.logspace(1, 10, num=4, base=np.array([2, np.e, 3, 10]))
Out[14]: array([2.000000e+00, 5.459815e+01, 2.187000e+03, 1.000000e+10])
```

Reply:
Done.